packages/app-cli/tests/html_to_md/table_with_empty_header.md

```diff
@@ -5,5 +5,6 @@
 | [Source](https://github.com/nim-lang/nim) | The github project |
 | [nimble](https://github.com/nim-lang/nimble) | The nim package manager |
 | [choosenim](https://github.com/dom96/choosenim) | Toolchain installer |
+|     |     |
```

Comment:
The HTML input file corresponding to this test contains
```html
		<tr>
			<td>&nbsp;</td>
			<td></td>
		</tr>
```
which was previously ignored.